apps/web/components/booking/BookingListItem.tsx

```diff
@@ -206,7 +219,7 @@ function BookingListItem(booking: BookingItemProps) {
       },
     });
   };
-
+  const showRecordingsButtons = isPast && isConfirmed && booking.location === "integrations:daily";
```

Comment:
```suggestion
  const showRecordingsButtons = booking.location === "integrations:daily" && isPast && isConfirmed;
```
With this change we leave early